bundle/edu.gemini.shared.gui/src/main/scala/edu/gemini/shared/gui/textComponent/NumberField.scala

```diff
@@ -19,8 +19,7 @@ object NumberField {
 
 class NumberField(d: Option[Double], allowEmpty: Boolean, format: java.text.Format = NumberField.df) extends FormattedTextField(format) with SelectOnFocus {
   d.orElse(Some(0)).foreach { d =>
-    import NumberField.df
-    text = df.format(d)
+    text = format.format(d)
```

Comment:
This was almost certainly a bug in the original code. The default formatter was always being used instead of the provided formatter.